src/App.js

```diff
@@ -1,34 +1,22 @@
-import { useState, useEffect, useContext } from "react";
+import { useState } from "react";
 import { searchRecipes } from "./api";
 import RecipeSearchForm from "./component/RecipeSearchForm";
 import RecipeList from "./component/RecipeList";
 import DisplayRecipe from "./component/DisplayRecipe";
 import Bookmark from "./component/Bookmark";
 import RecipeAddButton from "./component/RecipeAddButton";
 import Route from "./component/Route";
-import { NavigationContext } from "./context/NavigationContext";
+
+const RECIPE_PATH = /\/recipes\//;
```

Comment:
My initial idea is that this should be a string instead, and in `Route` component, this string will be parsed to a RegExp with `const re = new RegExp("/recipes/")`. Your approach is also fine, but for example if you gonna create new routes, you have to define more regex, which is not ideal imo 🤔